frontend/src/components/UserBookings.vue

```diff
@@ -1,27 +1,36 @@
 <template>
-  <div class="bg-gray-700 bg-opacity-50 rounded-3xl p-5 my-10" v-if="bookings.length">
+  <div class="bg-gray-700 bg-opacity-50 rounded-3xl py-10 px-20 my-10" v-if="bookings.length">
+    <h2 class="text-center mb-10">Bookings</h2>
+    <div
+        class="md:grid md:grid-cols-2 md:gap-4"
+        v-for="(booking, index) in bookings"
+        :key="booking.id"
+    >
+      <!-- Booking Details Column -->
+      <div>
+        <h3>{{ booking.schedule.movie.name }}</h3>
+        <p class="pt-3">
+          <strong>Cinema:</strong> {{ booking.schedule.cinema.name }} - {{ booking.schedule.cinemaHall.name }}
+        </p>
+        <p><strong>Duration:</strong> {{ booking.schedule.movie.duration }} minutes</p>
+        <p><strong>Average Rating:</strong> {{ averageRatings[booking.schedule.movie.id] || 'Loading...' }}%</p>
+        <p><strong>Showtime:</strong> {{ new Date(booking.schedule.localDateTime).toLocaleString().slice(0, 16) }}</p>
+        <p><strong>Booked Seats:</strong> {{ booking.bookedSeats.join(', ') }}</p>
+      </div>
 
-    <h2 class="text-center">Bookings</h2>
-    <div class="container mx-auto p-3 flex justify-evenly" v-for="(booking, index) in bookings" :key="booking.id">
-        <div>
-          <h4>{{ booking.schedule.movie.name }}</h4>
-          <h5 class="pt-3"><strong>Cinema:</strong> {{ booking.schedule.cinema.name }} -
-            {{ booking.schedule.cinemaHall.name }}</h5>
-          <p><strong>Duration:</strong> {{ booking.schedule.movie.duration }} minutes</p>
-          <p><strong>Average Rating:</strong> {{ averageRatings[booking.schedule.movie.id] || 'Loading...' }}%</p>
-          <p><strong>Showtime:</strong> {{ new Date(booking.schedule.localDateTime).toLocaleString() }}</p>
-          <p><strong>Booked Seats:</strong> {{ booking.bookedSeats.join(', ') }}</p>
-          <hr v-if="isLastBooking(booking)" class="mt-6">
-        </div>
+      <div class="flex justify-start md:justify-center mt-3 font-bold">
         <user-rating :movieId="booking.schedule.movie.id" @ratingSubmitted="updateAverageRating"></user-rating>
+      </div>
 
+      <!-- Divider Line After Each Booking Entry on Mobile -->
+      <div v-if="isLastBooking(booking)" class="col-span-1 border-t border-gray-500 my-6"></div>
```

Comment:
We should probably use the "old" hr tag to properly utilize the html semantic
```suggestion
      <hr v-if="isLastBooking(booking)" class="my-6 border-gray-500 ">
```

Reply:
Yes, you are right!